setup/agreement/setup.py

```diff
@@ -0,0 +1,6 @@
+import setuptools
```

Comment:
This should comes in a separated commit named  [MIG] agreement: pre-commit execution